dotnet/Simple.Sauce/SauceOptions.cs

```diff
@@ -56,19 +159,92 @@ public SauceOptions()
         public string TimeZone { get; set; }
         public string TunnelIdentifier { get; set; }
         public bool VideoUploadOnPass { get; set; }
-
-        public void WithEdge()
+        public DriverOptions SeleniumOptions { get; set; }
+        public Browser BrowserName { get; set; } = Browser.Chrome;
+        //TODO not fond of this name
+        public static List<string> W3CAllowedOptionsList = new List<string>(new string[]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=4945851)